config/apache-ignite-zos/ignite-log4j2.xml

```diff
@@ -30,8 +30,8 @@
         <Routing name="FILE">
             <Routes pattern="$${sys:nodeId}">
                 <Route>
-                    <RollingFile name="Rolling-${sys:nodeId}" fileName="${sys:IGNITE_HOME}/work/log/ignite-${sys:nodeId}.log"
-                                 filePattern="${sys:IGNITE_HOME}/work/log/ignite-${sys:nodeId}-%i-%d{yyyy-MM-dd}.log.gz">
+                    <RollingFile name="Rolling-${sys:nodeId}" fileName="${sys:IGNITE_HOME:-}/work/log/ignite-${sys:nodeId}.log"
```

Comment:
Does it mean it will try to log into /work by default (root folder)? Can we substitute it with `.' (cur dir)?